meetings/cpansec-minutes-2025-01-08.md

```diff
@@ -0,0 +1,157 @@
+    *   …
+4.  [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    *   New and existing documentation being discussed in relation to open subjects
+5.  [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+    *   …
+6.  [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+7.  [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    *   …
+8.  [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    *   …
+9.  [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+    *   …
+10. [CPAN Secure by Default](https://github.com/orgs/CPAN-Security/projects/15)
+    *   …
+11. [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+
+### TLS/HTTPS/CSPRNG/DSA in core
+- @leont - Worked on BearSSL & OpenSSL
+  - cleaning up CORE is required but tedious
+  - @sjn - is funding wanted/useful
+
+### Funding drive
+- [German Sovereign Tech fund](https://www.sovereign.tech/programs/fund)
+- [GitHub Secure Open Source Fund](https://resources.github.com/github-secure-open-source-fund/) deadline Jan 7th
+- Alpha-Omega
+- @sjn will work on that after FOSDEM - will need to know who is open to do (funded) work
+    - @stigtsp 1d/w
+
+### Ongoing vulnerabilities
+- @robrwo - POSIX::2008 CVE-2024-55564 already added for &lt; v0.24, not for &lt; v0.04
+    - @robrwo - Net::OAuth released with fix, CVE-2025-22376
+    - WWW::OAuth released with fix based on fix to Net::OAuth, CVE request by @robrwo
+    - Several modules waiting for ack from authors or 20/30 Jan to req CVEs and disclosure
+- @tux - SQLite CVE was closed in [CPANSA DB](https://metacpan.org/release/BDFOY/CPANSA-DB-20250104.001/source/Changes#L7)
+
+### Author guide to generating random values for security
+- @robrwo - Online
+   - Link posted on security.metacpan.org front page
+   - No feedback
+   - New version of Crypt::URandom (the main module recommended), but lots of issues with that
+   - Maybe promote it on blogs.perl.org by late Jan
+   - PR for Perl to change docs for recommended modules
+- @stigtsp - summarizing this work is useful/positive
+
+### CPAN Author's Security Policy Guidelines
+- @robrwo - Online
+  - Out of draft
+  - @robrwo posted about it on blogs.perl.org. That post, and a link to original guidelines posted by briandfoy on Reddit.
+  - 65 distributions on CPAN have a SECURITY.md as of 7 Jan
+  - PR for MetaCPAN to highlight the document like it does README, LICENSE etc
+  - Most important: we are getting feedback
+- @tux - Software::Security::Policy::Individual - do we need ::Team (or other name)?
+  - team policies to be documented: active teams, active groups, active Github projects with more maintainers but no single person to address for vulnerabilities (hidden Github security issues)
+
```

Comment:
- rename attribute to make its usage clear (support for perl versions, not he release at hand)
 ^^ does that need to be in the minutes?

Reply:
What should change in the minutes in this regard? (suggestions, please)

Reply:
I'll add "discussed also different ideas about how templates will evolve", based on @robrwo's feedback.